go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/algorand/go-codec/codec v0.0.0-20190507210007-269d70b6135d
 	github.com/algorand/go-deadlock v0.2.1
 	github.com/algorand/graphtrace v0.0.0-20201117160756-e524ed1a6f64
-	github.com/algorand/msgp v1.1.45
+	github.com/algorand/msgp v1.1.46
```

Comment:
I'm not sure why we have this here, since it's already identical to the master.

Reply:
I added it here before you introduced a separate PR for it.  Will rebase against master, which should cause this diff to go away.